### DISCUSSION TITLE
Merge member-manage and member-role

### DISCUSSION THREAD

**Discussion by WarnDa** [BG-Software-LLC/SuperiorSkyblock2#1106] [Pending Feature Requests] [answered]
Currently there are two menu to manage a player (change role, kick, ban) and if you want to change the role, it opens another gui.
So you got :
- 1 gui with 3 buttons (kick / ban / change role)
- 1 gui with each role

You can easily merge theses guis into one, it will make less guis, so easier for player to manager other members.

**Comment by Lemvs** [comment]
Added in latest dev build. You can add role buttons to the member-manage menu in the same way as in the member-role menu. However, we cannot remove this menu entirely, as that would break backward compatibility.

@OmerBenGera you can mark this answered and close it.

**Reply by Lemvs** [reply, marked as answer]
@OmerBenGera
